meetings/Agenda_2024-04-18.md

```diff
@@ -0,0 +1,20 @@
+## Meeting agenda April 18, 2024
+
+Topics to discuss:
+1. **Exec Check-in and debrief of year** (10-15 min)
+- Roses: What do you think went well this term? 
+- Thorns: What do you think we could have done better?
```

Comment:
- D: Gmail was hard to set up, access, learning how to use. Emailing has more reach and interaction - got people asking to be added to the mailing list. Didn't expect to take that long to learn. CDRS regulation fell short - Denny still didn't get key access. 
- V: Attendance - people sign up and don't come even online. Surved student body of what lessons they wanted and yet still had low attendance.